<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpeWidget</class>
 <widget class="QWidget" name="OpeWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1125</width>
    <height>929</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QHBoxLayout" name="horizontalLayout_29">
      <item>
       <widget class="QFrame" name="frame">
        <property name="styleSheet">
         <string notr="true">QFrame{
background-color: rgb(255, 255, 255);
border:none;
border-radius:10px;
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="1,10,0,0">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="frame_2">
           <property name="maximumSize">
            <size>
             <width>16777</width>
             <height>50</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame{
	
background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);
border-top-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_5">
              <property name="maximumSize">
               <size>
                <width>16777</width>
                <height>16777213</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QFrame{
border-top-left-radius:10px;
border-top-right-radius:10px;
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <property name="spacing">
                <number>18</number>
               </property>
               <property name="leftMargin">
                <number>18</number>
               </property>
               <property name="topMargin">
                <number>12</number>
               </property>
               <property name="rightMargin">
                <number>3</number>
               </property>
               <property name="bottomMargin">
                <number>18</number>
               </property>
               <item>
                <widget class="QPushButton" name="pushButton_5">
                 <property name="minimumSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	
	background-color: rgb(240, 108, 96);
border:1px solid rgba(113, 17, 15,50);
border-radius:6px;
}
QPushButton:hover {
background-color:rgb(139, 29, 31)
	
}
QPushButton:pressed {
	background-color: rgb(232, 59, 35);
}
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_6">
                 <property name="minimumSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	
	background-color: rgb(231, 193, 75);
border:1px solid rgb(166, 151, 66);
border-radius:6px;
}
QPushButton:hover {
	background-color: rgb(124, 111, 38);
	
}
QPushButton:pressed {
	
	background-color: rgb(231, 193, 75);
}
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="pushButton_7">
                 <property name="minimumSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                 <property name="maximumSize">
                  <size>
                   <width>12</width>
                   <height>12</height>
                  </size>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QPushButton{
	background-color: rgb(35, 158, 33);
border:1px solid rgb(25, 87, 16);
border-radius:6px;
}
QPushButton:hover {
	
	background-color: rgb(15, 118, 13);
	
}
QPushButton:pressed {
	background-color: rgb(35, 158, 33);
}
</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
               <item>
                <spacer name="horizontalSpacer">
                 <property name="orientation">
                  <enum>Qt::Horizontal</enum>
                 </property>
                 <property name="sizeHint" stdset="0">
                  <size>
                   <width>40</width>
                   <height>20</height>
                  </size>
                 </property>
                </spacer>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_3">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_8">
              <property name="styleSheet">
               <string notr="true">QFrame{
	background-color: rgb(237, 237, 237);
}
QFrame{
border-top-left-radius:0px;
border-top-right-radius:0px;
border-bottom-right-radius:0px;
border-bottom-left-radius:0px;
}</string>
              </property>
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,10">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QFrame" name="frame_10">
                 <property name="frameShape">
                  <enum>QFrame::StyledPanel</enum>
                 </property>
                 <property name="frameShadow">
                  <enum>QFrame::Raised</enum>
                 </property>
                 <layout class="QHBoxLayout" name="horizontalLayout_7">
                  <property name="spacing">
                   <number>5</number>
                  </property>
                  <property name="leftMargin">
                   <number>8</number>
                  </property>
                  <property name="topMargin">
                   <number>2</number>
                  </property>
                  <property name="rightMargin">
                   <number>8</number>
                  </property>
                  <property name="bottomMargin">
                   <number>2</number>
                  </property>
                  <item>
                   <widget class="QWidget" name="widget_3" native="true">
                    <property name="minimumSize">
                     <size>
                      <width>48</width>
                      <height>48</height>
                     </size>
                    </property>
                    <property name="maximumSize">
                     <size>
                      <width>48</width>
                      <height>48</height>
                     </size>
                    </property>
                    <property name="cursor">
                     <cursorShape>PointingHandCursor</cursorShape>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">QWidget{image:url(:/images/images/1.jpg);
border-radius:24px;
	background-color: rgb(255, 255, 255);
}</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="pushButton">
                    <property name="styleSheet">
                     <string notr="true">border:none;
text-align:left;</string>
                    </property>
                    <property name="text">
                     <string>jack</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </widget>
               </item>
               <item>
                <widget class="QScrollArea" name="scrollArea">
                 <property name="styleSheet">
                  <string notr="true">QScrollBar:vertical{
    width:8px;
    background:rgba(0,0,0,0%);

}
QScrollBar::handle:vertical{
    width:0px;
    background:rgba(125,125,125,20%);
    border-radius:4px;
}
QScrollBar::handle:vertical:hover{
    width:0px;
    background:rgba(125,125,125,70%);
    border-radius:4px;
}
QScrollBar::add-line:vertical{
    height:0px;width:0px;

}
QScrollBar::sub-line:vertical{
    height:0px;width:0px;

}
QScrollBar::add-line:vertical:hover{
    height:0px;width:0px;

}
QScrollBar::sub-line:vertical:hover{
    height:0px;width:0px;

}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(0,0,0,0%);
    border-radius:4px;
}</string>
                 </property>
                 <property name="widgetResizable">
                  <bool>true</bool>
                 </property>
                 <widget class="QWidget" name="scrollAreaWidgetContents">
                  <property name="geometry">
                   <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>1073</width>
                    <height>723</height>
                   </rect>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_3">
                   <property name="spacing">
                    <number>0</number>
                   </property>
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="bottomMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <widget class="QListWidget" name="m_pListW">
                     <property name="minimumSize">
                      <size>
                       <width>0</width>
                       <height>180</height>
                      </size>
                     </property>
                     <property name="maximumSize">
                      <size>
                       <width>16777215</width>
                       <height>180</height>
                      </size>
                     </property>
                     <property name="font">
                      <font/>
                     </property>
                     <property name="styleSheet">
                      <string notr="true">QListView {
	border-radius: 20px;
outline:0px;
}
QListView::item{
background-color: transparent;
	padding: 8px;
padding-left: 18px;
padding-right: 18px;
margin-bottom:0px;

}
QListView::item:hover {
background-color: rgba(216, 216, 216, 70);
	border-radius: 8px;



}
QListView::item:selected {
	/*background-color: transparent;*/
	border-radius: 8px;
background-color: rgba(216, 216, 216, 70);
	color: black;


}</string>
                     </property>
                     <property name="verticalScrollBarPolicy">
                      <enum>Qt::ScrollBarAlwaysOff</enum>
                     </property>
                     <property name="horizontalScrollBarPolicy">
                      <enum>Qt::ScrollBarAlwaysOff</enum>
                     </property>
                     <property name="iconSize">
                      <size>
                       <width>16</width>
                       <height>16</height>
                      </size>
                     </property>
                     <item>
                      <property name="text">
                       <string>好友</string>
                      </property>
                     </item>
                     <item>
                      <property name="text">
                       <string>网盘</string>
                      </property>
                     </item>
                    </widget>
                   </item>
                   <item>
                    <widget class="QFrame" name="frame_11">
                     <property name="frameShape">
                      <enum>QFrame::StyledPanel</enum>
                     </property>
                     <property name="frameShadow">
                      <enum>QFrame::Raised</enum>
                     </property>
                     <layout class="QVBoxLayout" name="verticalLayout_4">
                      <property name="spacing">
                       <number>0</number>
                      </property>
                      <property name="leftMargin">
                       <number>0</number>
                      </property>
                      <property name="topMargin">
                       <number>0</number>
                      </property>
                      <property name="rightMargin">
                       <number>0</number>
                      </property>
                      <property name="bottomMargin">
                       <number>0</number>
                      </property>
                     </layout>
                    </widget>
                   </item>
                  </layout>
                 </widget>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="QFrame" name="frame_7">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_23">
               <item>
                <widget class="QStackedWidget" name="stackedWidget">
                 <property name="styleSheet">
                  <string notr="true">${

	background-color: rgba(255, 255, 255, 0);
}


QScrollBar:vertical{
    width:8px;
    background:rgba(0,0,0,0%);

}
QScrollBar::handle:vertical{
    width:0px;
    background:rgba(125,125,125,20%);
    border-radius:4px;
}
QScrollBar::handle:vertical:hover{
    width:0px;
    background:rgba(125,125,125,70%);
    border-radius:4px;
}
QScrollBar::add-line:vertical{
    height:0px;width:0px;

}
QScrollBar::sub-line:vertical{
    height:0px;width:0px;

}
QScrollBar::add-line:vertical:hover{
    height:0px;width:0px;

}
QScrollBar::sub-line:vertical:hover{
    height:0px;width:0px;

}
QScrollBar::add-page:vertical,QScrollBar::sub-page:vertical
{
    background:rgba(0,0,0,0%);
    border-radius:4px;
}</string>
                 </property>
                 <property name="currentIndex">
                  <number>0</number>
                 </property>
                 <widget class="QWidget" name="page">
                  <layout class="QHBoxLayout" name="horizontalLayout_22"/>
                 </widget>
                 <widget class="QWidget" name="page_2"/>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_27">
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_12">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_4">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0.505682, x2:1, y2:0.477, stop:0 rgba(11, 131, 120, 219), stop:1 rgba(85, 98, 112, 226));
	color:rgba(255, 255, 255, 210);</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="2">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>12</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>12</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QFrame" name="frame_15">
              <property name="frameShape">
               <enum>QFrame::StyledPanel</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Raised</enum>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_16"/>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton_5</sender>
   <signal>clicked()</signal>
   <receiver>OpeWidget</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>35</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_7</sender>
   <signal>clicked()</signal>
   <receiver>OpeWidget</receiver>
   <slot>showMinimized()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>95</x>
     <y>33</y>
    </hint>
    <hint type="destinationlabel">
     <x>548</x>
     <y>464</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
